frontend/src/scenes/experiments/Metrics/ExperimentMetricModal.tsx

```diff
@@ -122,7 +122,7 @@ export function ExperimentMetricModal({
             </div>
             <ExperimentMetricForm
                 metric={metric}
-                handleSetMetric={({ newMetric }: { newMetric: ExperimentMetric }) => {
+                handleSetMetric={(newMetric) => {
                     setMetric({ metricIdx, metric: newMetric, isSecondary })
```

Comment:
This is probably fine, but I would try to avoid unnecessary closures.

Reply:
Fixed in f3e0fb687939d330a3645824cff6666efa1cac4a

Is that what you meant here?